src/plugins/data_views/server/routes/create_index_pattern.ts

```diff
@@ -79,6 +81,7 @@ const registerCreateDataViewRouteFactory =
           const savedObjectsClient = ctx.core.savedObjects.client;
           const elasticsearchClient = ctx.core.elasticsearch.client.asCurrentUser;
           const [, , { dataViewsServiceFactory }] = await getStartServices();
+          usageCollection?.incrementCounter({ counterName: path });
```

Comment:
While `counterType` and `incrementBy` are optional, you could add even more detail by using `counterType` as `error` or `legacyError` or something along those lines. 

There are a few plugins that use `counterType` to describe the stats, e.g. alerting [uses](https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/routes/lib/track_legacy_terminology.ts#L27-L31) 'legacyTerminology' to track legacy terms.
Here we could do something similar:
```
usageCollection?.incrementCounter({
        counterName: path,
        counterType: 'legacyError'
      });
```